packages/amazonq/src/app/inline/completion.ts

```diff
@@ -60,6 +63,31 @@ export class InlineCompletionManager implements Disposable {
         }
     }
 
+    public registerCustomization(client: LanguageClient) {
+        commands.registerCommand(
+            'aws.amazonq.connect',
```

Comment:
could this be something like `aws.amazonq.customization`? it makes me think its logging in

Reply:
also does it make sense to register the command with our internal commands imlementation?